File: Union of two arrays/solve.java
class Solution{
    public static int doUnion(int a[], int n, int b[], int m) 
    {
        HashSet<Integer> hs = new HashSet<>();
        for(int i=0; i<a.length; i++){
            hs.add(a[i]);
        }
        
        
        for(int j=0; j<b.length; j++){
            hs.add(b[j]);
        }
        
        return hs.size();
    }
}

File: Union of two arrays/solve.py
class Solution:    
    #Function to return the count of number of elements in union of two arrays.
    def doUnion(self,a,n,b,m):
        
        us = set()

        for i in range(n):
            us.add(a[i])

        for i in range(m):
            us.add(b[i])

        return len(us)
